lib/authz/permissions.go

```diff
@@ -585,7 +585,15 @@ func (a *authorizer) authorizeAdminAction(ctx context.Context, authContext *Cont
 	requiredExt := &mfav1.ChallengeExtensions{Scope: mfav1.ChallengeScope_CHALLENGE_SCOPE_ADMIN_ACTION}
 	mfaData, err := a.mfaAuthenticator.ValidateMFAAuthResponse(ctx, mfaResp, authContext.User.GetName(), requiredExt)
 	if err != nil {
-		return trace.Wrap(err)
+		// invalid MFA verification should be a noop, this way an actionable error will be
+		// returned instead. e.g. "admin-level API request requires MFA verification".
+		//
+		// TODO(Joerger): Currently we get a validation error when a reusable Webauthn response
+		// is provided but it's session data has been overwritten. This happens in bulk
+		// `tctl create` requests with endpoints mixed in that disallow reuse. Ignoring the error
+		// here is harmless, but a better way to handle this would be to add a separate index for
+		// reusable Webauthn session data.
```

Comment:
@codingllama Any thoughts on this? Right now we just have `login` and `register`. I think we'd just need to add `login_with_reuse`. When applicable, we would just check for `login`, `login_with_reuse`, or both. I could do this in a follow up PR if it sounds reasonable to you.

Reply:
I'm not sure I follow the suggestion. Isn't this what allow reuse is for? If we want to allow challenge reuse for some resources, shouldn't we do that instead?

Reply:
My concern is that even if we make all `tctl create` endpoints allow reuse now, we might add one in the future which doesn't, which would cause the flow to breakdown completely with no way around it (due to the issue explained here). I suppose we could just be careful not to screw it up and worst case do a patch fix if it happens.